challengers/Kristina_Gurung/hard-task-5-responsive-dashboard-sidebar/style.css

```
@@ -0,0 +1,25 @@
+body {margin:0; font-family: Inter,Segoe UI,Arial,sans-serif; background:#eef2f8; color:#111827;}
```

Stylelint (17.6.0)

[error] Expected quotes around "Segoe UI" (font-family-name-quotes)

(font-family-name-quotes)